tests/Integration/Subscriber/ExpiredCachePurgeSubscriber/TestCleanCacheScheduledEvent.php

```diff
@@ -0,0 +1,95 @@
+<?php
+namespace WP_Rocket\Tests\Integration\Subscriber\ExpiredCachePurgeSubscriber;
+
+use PHPUnit\Framework\TestCase;
```

Comment:
Change to [`WP_Rocket\Tests\Integration\TestCase`](https://github.com/wp-media/wp-rocket/blob/develop/tests/Integration/TestCase.php), which requires `develop` to be merged into this branch.